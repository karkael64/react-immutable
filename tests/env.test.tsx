import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import "global-jsdom/register";

const Comp: React.FC = () => {
  return <p>Test</p>;
};

test("env", () => {
  expect(test).toBeDefined();
});

test("env in component", () => {
  const { baseElement } = render(<Comp />);
  expect(baseElement).toMatchSnapshot();
});
