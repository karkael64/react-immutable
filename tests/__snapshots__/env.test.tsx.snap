// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`env in component 1`] = `
<body>
  <div>
    <p>
      Test
    </p>
  </div>
</body>
`;
